app/views/service_requests/cart/_cart_sub_service_requests.html.haml

```diff
@@ -29,13 +29,18 @@
         .list-group-item.line-item
           .row
             .col-10.d-flex.align-items-center.pr-0
-              %h6.mb-0{ class: ssr.is_complete? ? 'text-success' : ssr.is_locked? ? 'text-danger' : '' }<
+              %h6.mb-0{ class: ssr.is_complete? ? 'text-success' : ssr.is_locked? ? 'text-danger' : '' }
                 = li.service.abbreviation
                 - if li.service.cpt_code.present?
                   %strong
                     = "(#{li.service.cpt_code})"
                 - unless li.service.is_available
                   = inactive_tag
             - if li.optional? && ssr.can_be_edited? && ['document_management'].exclude?(action_name)
-              .col-2.pl-0.text-right
-                = link_to icon('fas', 'trash-alt'), remove_service_service_request_path(srid: service_request.id, line_item_id: li.id), remote: true, method: :delete, class: 'btn btn-sm btn-sq btn-danger remove-service', title: t('proper.cart.remove_service'), data: { toggle: 'tooltip', disable: true }
+              - unless ssr.in_work_fulfillment && (li == ssr.line_items.last)
```

Comment:
@6reg I double-checked this after our discussion and I think you need `(ssr.line_items.count == 1)` instead of `(li == ssr.line_items.last)` in your check here.